lua/conform/formatters/luaformatter.lua

```diff
@@ -0,0 +1,13 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/Koihik/LuaFormatter",
+    description = "Reformats your Lua source code.",
+  },
+  command = "lua-format",
+  args = { "-i", "$FILENAME" },
+  stdin = false,
```

Comment:
One quick check: does it format if you use `stdin = true` and don't pass any args?